app/assets/stylesheets/application/components/header.scss

```diff
@@ -230,3 +230,18 @@ body:not(.section-home) {
     padding: 1rem;
   }
 }
+
+// for active_material
```

Comment:
This sheet is not included in active_admin.scss, so these styles have no effect.

Reply:
Can we just remove them? The ticket is to fix the menu drop down, I'd rather now realign the rest of the menu as well.

Reply:
Strange, it worked in my sandbox!

I'm not sure what you mean by "just remove them"?

Reply:
These lines added here do nothing, because this spreadsheet is not used by active admin.

When you take these styles and move them into active_admin.scss, the admin menu looks like this:

![Screenshot_2020-02-27_09-38-43](https://user-images.githubusercontent.com/6469642/75476900-3fbf9f80-5950-11ea-8e21-5c22a03148c0.png)

The menu is realigned so that the links are horizontally centered, and the padding around the logout link gets kicked out of alignment as well. So when I say remove them I mean that since they do nothing here, they should be removed, and that if we moved them so that they would actually apply to the admin area, the menu would get screwed up -- so we should remove them in any case.

Unless there's something I'm missing about what these styles are supposed to do?

Please try deleting them to see that nothing changes.